app/uk/gov/hmrc/eoricommoncomponent/frontend/views/contact_address.scala.html

```diff
@@ -0,0 +1,123 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.eoricommoncomponent.frontend.views.helpers.subscription.ViewHelper
+@import uk.gov.hmrc.eoricommoncomponent.frontend.forms.models.{AddressViewModel, YesNoWrongAddress}
+@import views.html.helper._
+@import views._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.views.html._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.models.Service
+@import uk.gov.hmrc.govukfrontend.views.html.components.{GovukButton, GovukFieldset, GovukRadios, GovukInsetText, GovukHint}
+@import uk.gov.hmrc.govukfrontend.views.Aliases.{Button, Text}
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.fieldset.{Fieldset, Legend}
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.{Radios, RadioItem}
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.hint.Hint
+
+@this(
+layout_di: layout,
+inputRadioGroup: helpers.inputRadioGroup,
+govukButton: GovukButton,
+inputText: helpers.inputText,
+errorSummary: helpers.errorSummary,
+goveukFieldSet: GovukFieldset,
+govukRadios : GovukRadios,
+govukHint: GovukHint
+)
+
+
+@(contactAddress: AddressViewModel, isInReviewMode: Boolean, contactAddressDetailsYesNoAnswerForm: Form[YesNo], service: Service)(implicit request: Request[_], messages: Messages)
+
+@fieldsetHint = {
+@govukHint(Hint(id= Some("fieldset-hint"), content = Text(messages("cds.subscription.contact-details.intro")), classes = "govuk-!-margin-bottom-3"))
+}
+
+@labelForEUCountry(countryCode: String) = @{
+messages(messageKeyForEUCountryCode(countryCode))
+}
+
+@messageKeyForEUCountryCode(countryCode: String) = @{
+s"cds.country.$countryCode"
+}
+
+@isEUCountryCode(countryCode: String) = @{
+messages.isDefinedAt(messageKeyForEUCountryCode(countryCode))
+}
+
+@countryCodeToLabel(code: String) = @{
+code match {
+case "GB" => messages("cds.country.GB")
+case c if isEUCountryCode(c) => labelForEUCountry(c)
+case nonEuCode => nonEuCode
+}
+}
+
+@fourLineBusinessAddress = @{
+List(contactAddress.street,
+contactAddress.city,
+contactAddress.postcode.getOrElse(""),
+countryCodeToLabel(contactAddress.countryCode)
+).filter(_.nonEmpty)
+}
+
+@hintHtml = {
+<div class="govuk-inset-text" id="contact-address">@{Html(fourLineBusinessAddress.mkString("<br>"))}</div>
+}
+
+
+
+@radioGroupYes = @{
+messages("confirm-contact-address-details.yes")
+}
+
+@radioGroupNo = @{
+messages("confirm-contact-address-details.no")
+}
+
+@displayInputRadioGroup(field: String, options: Seq[(String, String)]) = {
+@inputRadioGroup(
+contactAddressDetailsYesNoAnswerForm(field),
```

Comment:
Can you change the alighment of these please?

Reply:
Alignment had been fixed now